PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice/src/main/java/com/microsoft/azure/toolkit/intellij/appservice/AppServiceIntelliJActionsContributor.java

```diff
@@ -0,0 +1,154 @@
+import com.microsoft.azure.toolkit.intellij.legacy.appservice.action.SSHIntoWebAppAction;
+import com.microsoft.azure.toolkit.intellij.legacy.appservice.action.StartStreamingLogsAction;
+import com.microsoft.azure.toolkit.intellij.legacy.appservice.action.StopStreamingLogsAction;
+import com.microsoft.azure.toolkit.intellij.legacy.function.action.CreateFunctionAppAction;
+import com.microsoft.azure.toolkit.intellij.legacy.function.action.DeployFunctionAppAction;
+import com.microsoft.azure.toolkit.intellij.legacy.webapp.action.CreateWebAppAction;
+import com.microsoft.azure.toolkit.intellij.legacy.webapp.action.DeployWebAppAction;
+import com.microsoft.azure.toolkit.lib.Azure;
+import com.microsoft.azure.toolkit.lib.appservice.AzureFunction;
+import com.microsoft.azure.toolkit.lib.appservice.AzureWebApp;
+import com.microsoft.azure.toolkit.lib.appservice.entity.FunctionEntity;
+import com.microsoft.azure.toolkit.lib.appservice.model.AppServiceFile;
+import com.microsoft.azure.toolkit.lib.appservice.service.IAppService;
+import com.microsoft.azure.toolkit.lib.appservice.service.impl.FunctionApp;
+import com.microsoft.azure.toolkit.lib.appservice.service.impl.WebApp;
+import com.microsoft.azure.toolkit.lib.appservice.service.impl.WebAppDeploymentSlot;
+import com.microsoft.azure.toolkit.lib.common.action.Action;
+import com.microsoft.azure.toolkit.lib.common.action.ActionView;
+import com.microsoft.azure.toolkit.lib.common.action.AzureActionManager;
+import com.microsoft.azure.toolkit.lib.common.bundle.AzureString;
+import com.microsoft.azure.toolkit.lib.common.entity.IAzureBaseResource;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTask;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Optional;
+import java.util.function.BiConsumer;
+import java.util.function.BiPredicate;
+
+public class AppServiceIntelliJActionsContributor implements IActionsContributor {
+    public static final int INITIALIZE_ORDER = AppServiceActionsContributor.INITIALIZE_ORDER + 1;
+    private static final String UNABLE_TO_OPEN_EXPLORER = "Unable to open explorer";
+
+    @Override
+    public void registerActions(AzureActionManager am) {
+        final BiConsumer<AppServiceFile, AnActionEvent> openFileHandler = (file, e) -> AzureTaskManager
+                .getInstance().runLater(() -> new AppServiceFileAction().openAppServiceFile(file, e.getProject()));
+        final ActionView.Builder openFileView = new ActionView.Builder("Open File", null)
+                .title(s -> Optional.ofNullable(s)
+                        .map(r -> AzureString.format("appservice|file.download", ((AppServiceFile) r).getName()))
+                        .orElse(null))
+                .enabled(s -> s instanceof AppServiceFile);
+        am.registerAction(AppServiceFileActionsContributor.APP_SERVICE_FILE_VIEW, new Action<>(openFileHandler, openFileView));
```

Comment:
keep action id as action + <operation id>

Reply:
Will resolve in later PR